strongbox-db-liquibase/pom.xml

```
@@ -8,7 +8,7 @@
     <parent>
         <groupId>org.carlspring.strongbox</groupId>
         <artifactId>strongbox-db</artifactId>
-        <version>1.0-SNAPSHOT</version>
+        <version>1.0-PR-16-SNAPSHOT</version>
```

Version is incorrect - should be `18`
```suggestion
        <version>1.0-PR-18-SNAPSHOT</version>
```

---

fixed